go/general: comments for disapprove (#1223)
exercism/website-copy: automated-comments/go/general/syntax_error.md

> Unfortunately there is a syntax error in this solution. Try running the tests in a terminal with `go test`.

Reviewer: ```suggestion
Unfortunately there is a syntax error in this solution. Run `go test` in a terminal.
```

Reviewer 2: To me, it looks a little bit like the student might think that running `go test` might solve the syntax error :D

Maybe something like:

```suggestion
Unfortunately there is a syntax error in this solution. This causes the tests to fail, which can be verified by running the tests in a terminal with `go test`.
```

I'm not sure how to elegantly word the action a student should take (fix the syntax error).

Author: Maybe in this casy it is obvious ;)
